docs/targets/solana.rst

```diff
@@ -183,6 +183,25 @@ this. The balance of an account can be credited without any code being executed.
 
 ``receive()`` functions are not permitted on the Solana target.
 
+.. _msg_sender_solana:
+
+``msg.sender`` not available on Solana
+______________________________________
+
+On Ethereum, ``msg.sender`` is used to identify the account which either submitted
+the transaction, or the caller when one contract calls another. On Ethereum, each
+contract execution involves a single contract. On Solana, each contract execution
```

Comment:
`contract execution involves a single contract`
Is this what you wanted to say?